### PR TITLE
chore: [CO-413] maven build

### PR DESCRIPTION
**Pull request by frisonisland**
Maven build using new mailbox parent pom.

- [x] Change carbonio-build to use maven target on lookup-store (https://github.com/Zextras/carbonio-build/pull/34)

### CODE REVIEW

File: ivy.xml [hunk 1]
```diff
@@ -9,36 +9,6 @@ SPDX-License-Identifier: AGPL-3.0-only
 <ivy-module version="2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
```
Comment (keshavbhatt) [inline ivy.xml]: Is this file still required?

**Review by keshavbhatt — APPROVED** [review]
Please check if ivy.xml is still relevant.

**Review by aheeva-yuliya — APPROVED** [review]

File: ivy.xml (hunk 1 above)
Comment (frisonisland) [reply ivy.xml]: I removed it, it is not required, thanks
